e2e/test/scenarios/visualizations-charts/maps.cy.spec.js

```diff
@@ -206,7 +206,7 @@ describe("scenarios > visualizations > maps", () => {
     // Ensure chart is rendered
     cy.get(".leaflet-interactive");
 
-    cy.findByTestId("viz-type-button").click();
+    openVizType(null);
```

Comment:
this is fixed in a followup PR. It was used like this because of a Map visualization which didn't have sections

https://github.com/metabase/metabase/pull/49928/files#diff-26ad07af1fbc2d37d9b9cfbbedd2b9602469cf4d00f220ce3baf4737ec6f9268R78